### DISCUSSION TITLE
I added my entire root directory to Git and can't figure out how to undo it

### DISCUSSION THREAD

**Discussion by BryanNilsen** [nss-day-cohort-47/C47-Discussions#16] [Help Me] [answered]
**Name:** Bryan Nilsen

**Have you searched the other help tickets?**
There aren't any others at this time :)

**Describe your issue**
I somehow managed to commit my ENTIRE workspace directory to git. I can't figure out how to un-git it. (for lack of a better term)

**List any error messages that appear**
lots of fatal error messages

**What have you googled?**
https://www.freecodecamp.org/news/how-to-undo-a-git-add/

https://stackoverflow.com/questions/2845731/how-to-uncommit-my-last-commit-in-git

https://scotch.io/tutorials/how-to-use-git-remove-without-deleting-the-files

**What have you tried?**
git rm -r --cached <folder>
git reset
git reset --soft HEAD^

**Comment by BryanNilsen** [comment, marked as answer]
try `rm -rf .git` in project directory that you want to remove git tracking from

**Reply by BryanNilsen** [reply]
hey that worked

**Reply by brendalong** [reply]
That totally saved my life. Thanks.

**Reply by heymonicakay** [reply]
I was literally JUST trying to fix this.. you just saved me like, hours of Googling 😅

**Reply by vishus12** [reply]
I actually had this problem and rm -rf .git solved it 😅😅😅😅
